extrapolation.py

```diff
@@ -591,6 +591,14 @@ def extrapolate(**kwargs):
           outside of the input data, it will be reset to the midpoint
           of the waveform: (W_outer.T(0)+W_outer.T(-1))/2
 
+        ErrorTol                 None
+          ONLY USED FOR Psi1 AND Psi0 WAVEFORMS.
+          ErroTol is the effective noise floor of the simulation.
+          If Psi1 or Psi0 (NOT scaled by radius) falls beneath
+          this tolerance for certain extraction radii, then those
+          radii are not included in the extrapolation. The value
+          1.0e-9 seems to work well for a few BBH systems.
+
```

Comment:
I didn't understand what this meant until I read the code.  Calling it a tolerance makes me think that the code will run some procedure until it achieves some level of accuracy, and makes me think that it might be a fractional rather than an absolute value.  I think renaming this variable `NoiseFloor` and removing the word "tolerance" would make it clearer.